controllers/html/build-routes.js

```diff
@@ -69,15 +80,19 @@ router.get('/:id', (req, res) => {
 						return resObj;
 					});
 			}),
-
-
-
 		//chain 2
 		mtg.card.where(whereObj)
+		
 	])
 		.then(([dbDeckData, cards]) => {
 			// console.log(dbDeckData);
-			// console.log(cards);
+			console.log(cards);
+			const id_arr = dbDeckData.deck_components.map(card => card.multiverseId);
+			console.log(id_arr);
+			dbDeckData.cards = dbDeckData.cards.map(card => {
+				card.count = id_arr.filter(id => card.multiverseid == id ).length;
+				return card;
+			})
```

Comment:
i had a bug on line 92: cannot read property map of undefined.

`dbDeckData.deck_components = dbDeckData.deck_components.map(card => {`

fixed it.

Reply:
that fix did indeed cause errors. instead, replace lines 73-74 with

`if (resObj.deck_components.length == 0) {
	resObj.cards = [];
	return resObj;
}
`